stable/memcached/templates/statefulset.yaml

```diff
@@ -19,6 +19,10 @@ spec:
         heritage: "{{ .Release.Service }}"
     spec:
       affinity:
+{{- if .Values.nodeAffinity }}
```

Comment:
I'd suggest to refactor the whole affinity configuation following this example:

https://github.com/kubernetes/charts/blob/302ed5bf18c4fa5017c3349465627b8ed51bf47e/incubator/vault/templates/deployment.yaml#L112
https://github.com/kubernetes/charts/blob/302ed5bf18c4fa5017c3349465627b8ed51bf47e/incubator/vault/values.yaml#L58-L67